symbology/custom-dictionary-style/src/main/java/com/esri/samples/custom_dictionary_style/CustomDictionaryStyleSample.java

```diff
@@ -67,31 +91,124 @@ public void start(Stage stage) {
       // set the initial viewpoint to the Esri Redlands campus
       map.setInitialViewpoint(new Viewpoint(new Point(-1.304630524635E7, 4036698.1412000023, map.getSpatialReference()), 5000));
```

Comment:
let's split this up over two lines for readability
```suggestion
      map.setInitialViewpoint(
      new Viewpoint(new Point(-1.304630524635E7, 4036698.1412000023, map.getSpatialReference()), 5000));
```